py/torch_tensorrt/dynamo/lowering/passes/_aten_lowering_pass.py

```diff
@@ -23,6 +24,7 @@
         fuse_prims_broadcast,
         replace_max_pool_with_indices,
         view_to_reshape,
+        propagate_shapes,
```

Comment:
Do we need this for any specific thing ? The shape info is already present in the graph (in node metadata). Can we re-use that ?